### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Customize New Project: The tooltip for "...(Browse)" button is given as "shift+Enter"

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3314]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.1.1; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.1.1

**NOTE: Same issue is observed in all the screens where "..."(browse) control is present** 

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Check if the tooltip for "...(Browse)" button after "Maven Home Directory" combo-box is proper

**Actual Result:**
The tooltip for "...(Browse)" button is given as "shift+Enter"

**Expected Result:**
The tooltip for "...(Browse)" button should be "Browse-shift+Enter"

**User Impact:**
Users will not understand the functionality of the button if name is not defined properly

**MAS Reference:**
[MAS 1.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})
[MAS 4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

![Bug_1_IntelliJ_Customize_Tooltip](https://user-images.githubusercontent.com/41108985/62203091-251c4c80-b3a8-11e9-83dd-4b443bcab20f.png)

**Comment by Flanker32** [comment]
This view belong to IntelliJ which is out of the scope of Azure Toolkits, we have created an issue in jetbrains to track this tooptip, here is the link https://youtrack.jetbrains.com/issue/IDEA-221593

**Comment by Flanker32** [comment]
- Importance of experience: Low, it's not a common case to change the maven home directory when create maven projects.
- Quality of experience: not impacted: Low, users may get a bit confused about the tooltip, but they could know by click the button and intellij will pop up the browser with title `Select Maven Home Directory`
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@soumyatippapuram  Please have a review,

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us verify it.

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749754
